templates/chrony.keys.epp

```diff
@@ -0,0 +1,6 @@
+<% if $chrony::chrony_password and $chrony::chrony_password != 'unset' { -%>
+<%= $chrony::commandkey %> <%= $chrony::chrony_password %>
+<% } -%>
+<% $chrony::keys.each |$line| { -%>
```

Comment:
are keys guaranteed to be at least `[]`?

Reply:
Good question. Yes, the required type defined in init.pp for $keys is Array[String[1]]